rebar.config

```diff
@@ -1,4 +1,8 @@
-{deps, [cargo]}.
+{deps, [
+    {cargo, ".*",
+        {git, "https://github.com/lrascao/erlang-cargo.git",
+            {branch, "feature/target_dir"}}}
```

Comment:
this branch is perched on top of https://github.com/rusterlium/erlang-cargo/pull/6 and will be PR'd after it